nodelet/src/loader.cpp

```diff
@@ -322,13 +322,13 @@ bool Loader::load(const std::string &name, const std::string& type, const ros::M
 
 
     ROS_DEBUG("Done initing nodelet %s", name.c_str());
-  } catch(...) {
+  } catch(std::runtime_error& e) {
```

Comment:
what about other possible exceptions (although probably not likely)?
Maybe catch `std::exception` instead?

Reply:
You're right, but in this file, `std::runtime_error` is caught  the same way in another location. If I changed it there, I'd have to change it at the other location for the sake of consistency.

The ultimately best way would be:
```
try {
    //
} catch (const std::exception& e) {
    // print error with e.what()
} catch (...} {
    // print error without e.what() in case something not derived from std::exception was thrown
}
```

Reply:
Changed both to `const std::exception&` now, but without the default catch.